src/node_api.h

```diff
@@ -65,11 +54,32 @@ typedef struct napi_module {
   NAPI_MODULE_INITIALIZER_X(NODE_API_MODULE_GET_API_VERSION_BASE,              \
                             NAPI_MODULE_VERSION)
 
+#define NODE_API_MODULE_SET_VTABLE                                             \
+  NAPI_MODULE_INITIALIZER_X(NODE_API_MODULE_SET_VTABLE_BASE,                   \
+                            NAPI_MODULE_VERSION)
+
+#ifdef NODE_API_MODULE_USE_VTABLE
+#define NODE_API_MODULE_SET_VTABLE_DEFINITION                                  \
+  const node_api_module_vtable* g_node_api_module_vtable =                     \
```

Comment:
Could we add a test that contains two compilation units and this global variable can be properly defined?